internal/cli/cmd/cluster/exposelb.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+package cluster
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8s "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"namespacelabs.dev/foundation/internal/cli/fncobra"
+	"namespacelabs.dev/foundation/internal/console"
+	"namespacelabs.dev/foundation/internal/fnerrors"
+	"namespacelabs.dev/foundation/internal/providers/nscloud/api"
+	"namespacelabs.dev/foundation/internal/providers/nscloud/ctl"
+	"namespacelabs.dev/foundation/std/tasks"
+)
+
+func NewExposeLoadBalancerCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "expose-lb [cluster-id]",
```

Comment:
what about:
`expose kubernetes`
`expose container`

and `expose` would be an alias of `expose container`